### PR TITLE
Fixed few UI tests

### PR DESCRIPTION
**Pull request by sghai**
- fixed gpg_key_edit locator on product and repo details page
- blocked gpgkey test with bz-id '1204602'
- fixed content-view locator to locate created CV name
- fixed manifest upload button locator

### CODE REVIEW

**Comment by sghai** [comment]
This PR needs to be merged before running the next automation run to have more fixes.

**Comment by omaciel** [comment]
ACK

File: tests/foreman/ui/test_gpgkey.py [hunk 1]
```diff
@@ -251,6 +251,7 @@ def test_positive_update_1(self):
             self.assertTrue(self.gpgkey.wait_until_element
                             (common_locators["alert.success"]))
 
+    @skip_if_bug_open('bugzilla', 1204602)
```
Comment (elyezer) [inline tests/foreman/ui/test_gpgkey.py]: Please add `@bz` testimony tag on the docstring.

File: tests/foreman/ui/test_gpgkey.py [hunk 2]
```diff
@@ -292,6 +293,7 @@ def test_positive_update_3(self):
...
```
Comment (elyezer) [inline tests/foreman/ui/test_gpgkey.py]: Please add `@bz` testimony tag on the docstring.

**Comment by elyezer** [comment]
ACK, pending comment.

**Comment by elyezer** [comment]
ACK

**Pull request merged**